index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portf Rod. (Mobile Main)</title>
+  </head>
+  <body>
+    <header id="header">
+      <div id="name">Rod.</div>
+      <a href="#">
+        <img id="hamburg" src="./image/Union.png" alt="" />
+      </a>
+    </header>
+    <main id="main">
+      <h1>
+        I'm Rod.
+        <br />
+        Glad to see you!
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product, feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h4>LET'S CONNECT</h4>
+      <section id="icons">
+        <img src="./image/social media icons.png" alt="" />
+      </section>

```

Comment:
- [ ] I would suggest you to not use one image for the social icon group as later in the future you will be adding links to every single one of those icons. Each icon in the social media icon groups should be individually placed and styled to make it look like[ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=ZYfC3oNgK3tRuTg3-0) . 
- [ ] Also please use list tags (`ul` and `li`)to in-list the social media icons and inclose each icon into a hyperlink tag (`<a>`) . Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.